pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -1449,6 +1449,9 @@ private boolean forceLog(BrokerResponse brokerResponse, long totalTimeMs) {
   @VisibleForTesting
   static void setOptions(PinotQuery pinotQuery, long requestId, String query, JsonNode jsonRequest) {
     Map<String, String> queryOptions = new HashMap<>();
+    // TODO: Remove the SQL query options after releasing 0.11.0
```

Comment:
Let's put it in the end (just before setting the query options) to ensure they don't get overridden. The query engine will break if these 2 options are missing during version upgrade